pywbem_mock/_baseprovider.py

```diff
@@ -41,34 +41,18 @@
 from __future__ import absolute_import, print_function
```

Comment:
The module docstring is shown in the docs and thus should be improved in these areas:
1. Line 25: Change "both the MainProvider, InstanceWriteProvider, and" to "both built-in providers and".
2. Line 29: There is no Method repository anymore. Beyond that, I think that list there should be rewritten.
3. Line 37: I don't understand this sentence. Remove?

Reply:
Rewritten.